DotNetHowTo/DotNetHowTo/Program.cs

```diff
@@ -215,26 +490,26 @@ private static void RunQueries(ISearchIndexClient indexClient)
             SearchParameters parameters;
             DocumentSearchResult<Hotel> results;
 
-            Console.WriteLine("Search the entire index for the term 'budget' and return only the hotelName field:\n");
+            Console.WriteLine("Search the entire index for the term 'motel' and return only the HotelName field:\n");
 
             parameters =
                 new SearchParameters()
                 {
-                    Select = new[] { "hotelName" }
+                    Select = new[] { "HotelName" }
                 };
 
-            results = indexClient.Documents.Search<Hotel>("budget", parameters);
+            results = indexClient.Documents.Search<Hotel>("motel", parameters);
 
             WriteDocuments(results);
 
-            Console.Write("Apply a filter to the index to find hotels cheaper than $150 per night, ");
+            Console.Write("Apply a filter to the index to find hotels with a room cheaper than $100 per night, ");
             Console.WriteLine("and return the hotelId and description:\n");
 
             parameters =
                 new SearchParameters()
                 {
-                    Filter = "baseRate lt 150",
-                    Select = new[] { "hotelId", "description" }
+                    Filter = "Rooms/any(s: s/BaseRate lt 100)",
```

Comment:
nit: It's idiomatic to name the lambda variable with the first letter of the collection, in this case `r` for "room" instead of `s`. Or you could just say `room`.

Reply:
Changed to r